src/Error/ExceptionRenderer.php

```diff
@@ -184,7 +184,6 @@ public function render()
                 'format' => 'array',
                 'args' => false
             ]);
-            $viewVars['_serialize'][] = 'trace';
```

Comment:
You still get a response with this change as well. Just not the stack trace. The stacktrace will still be in the logs.